docs/development/getting_started.rst

```diff
@@ -17,7 +17,7 @@ your development workstation.
 Ubuntu/Debian
 ~~~~~~~~~~~~~
 
-.. note:: Tested on: Ubuntu 14.04
+.. note:: Tested on: Ubuntu 14.04 and Debian sid (May 6th)
```

Comment:
> do you think Debian sid is too shifting a point of reference and just shouldn't be added?

This resonates with me. Your inclination to add a timestamp to a rapidly changing distro is good. Maybe we can compromise with `Debian Sid (Q2 2016)` for a broader window. Even that strikes me as rather narrow, though—will we update it again in Q3 2016?

Another troubling point is that the docs claim our setup has been tested on Ubuntu 14.04 LTS. That _was_ true, but I believe I was the most active Ubuntu user, and I'm now primarily developing on Debian. I do have an Ubuntu 16.04 box that I can run through the environment setup on and confirm it works as expected.